src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1104,6 +1104,13 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         if ($_.Exception -and $_.Exception.WasThrownFromThrowStatement) {
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter, exit, and re-enter another PSHost
```powershell
Expected $true, because The script was able to re-enter another process and grab the pid of '3376'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 96
96:                 Should -BeTrue -Because "The script was able to re-enter another process and grab the pid of '$pwshId'."
```